doc/api/fs.md

```diff
@@ -1682,7 +1686,8 @@ changes:
   * `start` {integer}
   * `end` {integer} **Default:** `Infinity`
   * `highWaterMark` {integer} **Default:** `64 * 1024`
-* Returns: {fs.ReadStream}
+  * `fs` {Object|null} **Default:** `null`
+* Returns: {fs.ReadStream} See [Readable Streams][].
```

Comment:
```suggestion
* Returns: {fs.ReadStream} See [Readable Stream][].
```

This will match the definition you added to the bottom and be consistent with the non-plural `[Writable Stream][]` link.